build.sh

```diff
@@ -0,0 +1,2 @@
+rm *.gem
```

Comment:
You don't need this file, you just have to run `rake build` to build the gem. And you can find all the available tasks by running `$ rake -T`:

```
rake bootstrap:update  # Update to a new version of Twitter Bootstrap
rake build             # Build bootstrap-generators-3.3.1.gem into the pkg directory
rake install           # Build and install bootstrap-generators-3.3.1.gem into system gems
rake release           # Create tag v3.3.1 and build and push bootstrap-generators-3.3.1.gem to Rubygems
rake test              # Run tests
```